### PR TITLE
Populate participants dialog

### PR DESCRIPTION
**Pull request by BarbaMatei**

### CODE REVIEW

**Review by Mihai-Sarmaru-Yonder — APPROVED** [review]
Backend looks good!

File: src/app/components/event-page/event-page.component.ts [hunk 1]
```diff
@@ -117,23 +119,24 @@ export class EventPageComponent implements OnInit {
...
-		{ id: 3, name: 'Radahan', email: 'abc', password: 'asdcasdcas' },
-		{ id: 4, name: 'Godfrey', email: 'abc', password: 'asdcasdcas' },
-		{ id: 5, name: 'Michael', email: 'abc', password: 'asdcasdcas' },
-	];
-
 	openParticipantsDialog(): Observable<boolean> {
+		var eventParticipants: UserDetails[] = [];
+		var isHost: boolean;
+
+		this.participantsService.getParticipant(this.event.id).subscribe(
```
Comment (larisaChh) [inline src/app/components/event-page/event-page.component.ts]: in the ngOnInit we already have this.participantsArray initialized. You should make use of that.

Comment (larisaChh) [inline src/app/components/event-page/event-page.component.ts:126]: in the ngOnInit we already have this.participantsArray initialized. You should make use of that.

File: src/app/services/participants-attendance.service.ts [hunk 2]
```diff
@@ -13,10 +14,16 @@ export class ParticipantsAttendanceService {
     //to-do:
     //once the attendance table is merged into master,come back here and configure the right links,
     //data type for the participants/attendance
+
+	getParticipant(eventId: number): Observable<UserDetails[]>{
```
Comment (larisaChh) [inline src/app/services/participants-attendance.service.ts]: We already have AttendanceService. That should be used and this service file deleted

Comment (larisaChh) [inline src/app/services/participants-attendance.service.ts:18]: We already have AttendanceService. That should be used and this service file deleted

File: api/Controllers/UserController.cs [hunk 3]
```diff
@@ -37,6 +37,12 @@ public async Task<IActionResult> GetUserById(int userId)
 			return Ok(_mapper.Map<UserDetailsDto>(user));
 		}
 
+		[HttpGet("[action]/{eventId}")]
+		public async Task<IActionResult> GetParticipantsForEvent(int eventId){
```
Comment (larisaChh) [inline api/Controllers/UserController.cs]: Why do we need this here if we already have it in the AttendanceController?

Comment (larisaChh) [inline api/Controllers/UserController.cs:41]: Why do we need this here if we already have it in the AttendanceController?

File: api/Service/UserService.cs [hunk 4]
```diff
@@ -35,6 +34,15 @@ public async Task<IList<User>> GetUserList()
 			return userList;
 		}
 
+		public async Task<IList<User>> GetParticipantsForEvent(int eventId){
```
Comment (larisaChh) [inline api/Service/UserService.cs:37]: This is a duplicate of        Task<IList<Attendance>> GetParticipants(int EventId); from AttendanceService. We do not need it twice.

Comment (larisaChh) [inline api/Service/UserService.cs]: This is a duplicate of        Task<IList<Attendance>> GetParticipants(int EventId); from AttendanceService. We do not need it twice. Remove this and use the one from the attendance controller

File: src/app/models/constants.ts [hunk 5]
```diff
@@ -29,6 +29,7 @@ export class RouteEndpoints {
     static readonly ATTENDANCE = 'Attendance'
     static readonly ATTENDANCE_GET_PARTICIPANTS = 'Attendance/GetParticipants'
     static readonly DASHBOARD_EVENTS = 'Event/GetDashboardEvents'
+    static readonly PARTICIPANTS_LIST = 'User/GetParticipantsForEvent'
```
Comment (larisaChh) [inline src/app/models/constants.ts:32]: remove this, make use of ATTENDANCE_GET_PARTICIPANTS = 'Attendance/GetParticipants'

Comment (larisaChh) [inline src/app/models/constants.ts]: remove this, make use of ATTENDANCE_GET_PARTICIPANTS = 'Attendance/GetParticipants'

**Pull request merged**